### PR TITLE
Changes needed for Github Actions

### PR DESCRIPTION
**Pull request by nikita-mk**
Signed-off-by: nikita-mk <nikkikokitkar@gmail.com>

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
I bit of details why are these needed would be helpful.

**Comment by geekodour** [comment]
can the `GITHUB_WORKSPACE` env var be used here?

https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/#environment-variables

In the `Makefile` you can add something like:
```
PROMBENCH_PATH             ?= ./
.
.
.
-f ${PROMBENCH_PATH}/manifests/prombench/nodepools.yaml
```

then you can probably set `PROMBENCH_PATH` to `/github/workspace` in the env var. not 100% sure though.

**Comment by krasi-georgiev** [comment]
yep that would be a much better approach, but I still fail to understand why running a docker container in github actions requires changing the paths.

**Comment by geekodour** [comment]
Continuation to the previous message, my assumption was wrong because `GITHUB_WORKSPACE` will be the `prometheus/prometheus` when we need `prometheus/prombench` cloning the repo beforehand and placing it in a shared directory will solve this issue.

**Comment by nikita-mk** [comment]
https://developer.github.com/actions/creating-github-actions/accessing-the-runtime-environment/#filesystem
Github actions has `/github/workspace` as working directory. It contains copy of the repository that triggered the workflow.

File: cmd/prombench/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,8 @@ RUN \
 
 COPY prombench /prombench/prombench
 COPY Makefile /prombench/Makefile
+COPY Makefile.common /prombench/Makefile.common
 COPY manifests/prombench /prombench/manifests/prombench
 
 WORKDIR /prombench
+ENTRYPOINT [ "make", "-C","/prombench/","deploy"]
```
Comment (nikita-mk) [inline cmd/prombench/Dockerfile:20]: The `ENTRYPOINT` instruction gets executed in `/github/workspace`.

Contents of the Dockerfile are present in `/prombench` dir.
Setting PROMBENCH_PATH is a good idea. We just need to pass this as an env var. I will test this

**Comment by krasi-georgiev** [comment]
The problem is that the workdir is set to `GITHUB_WORKSPACE` which is `/github/workspace` and it overwrites the default prombench workdir which is `/prombench`

to overcome this problem you use this
```
action "start_benchmark" {
  uses = "docker://prombench/prombench:2.0.1"
  runs = "bash"
  args = ["-c", "cd /prombench && make deploy"]
}
```

this way there would be no need to make any changes to the prombench image and just add the missing `Makefile.common`

**Comment by nikita-mk** [comment]
I tested this and it is worked.

**Comment by krasi-georgiev** [comment]
If it worked do you prefer to keep the PR as is or will update it?

**Comment by nikita-mk** [comment]
Actually, the only change we need is copying Makefile.common. That is being solved in another PR. 
I will close this PR

**Comment by nikita-mk** [comment]
See #228 for parameters to be given as file paths

**Pull request closed without merging**